membership/fixtures/membership_fees.json

```diff
@@ -138,5 +138,15 @@
         },
         "model": "membership.fee",
         "pk": 14
+    },
+    {
+        "fields": {
+            "start": "2016-01-01T00:00:01",
```

Comment:
maybe "start": "2016-10-15T16:08:01", is better timestamp for start point of new membership class